src/common/main.lua

```diff
@@ -450,6 +452,37 @@ function ant.init()
 			SourceCodeTxId = srcCodeTxId
 		end
 	)
+
+	-- IO Network Contract Handlers
+	Handlers.add(camel("Release-Name"), utils.hasMatchingTag("Action", "Release-Name"), function(msg)
```

Comment:
Can add this to the action map

Reply:
ah - added it, but forgot to come back and use it. [b07e874](https://github.com/ar-io/ar-io-ant-process/pull/23/commits/b07e874334d377ba35bb2056c4b9dbdd974a976f)